xtable-core/pom.xml

```diff
@@ -206,6 +218,14 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
Is this a requirement for Paimon? We're only publishing to a java 8 target currently

Reply:
I'm getting these compatibility errors when using java 8 on my machine, so I thought xtable was on 11 already but wasn't configured 

<img width="1105" height="161" alt="image" src="https://github.com/user-attachments/assets/0c9d224a-a729-4381-ae40-4e89eb8e265b" />

maybe that's an issue with my intellij, as it runs fine with 
<img width="1639" height="96" alt="image" src="https://github.com/user-attachments/assets/3d25d9f3-b1fe-4548-806b-756af482bf71" />

everything works fine on 11 🤔

Reply:
Yes we use Java 11 as the source and when [developing](https://github.com/apache/incubator-xtable?tab=readme-ov-file#building-the-project-and-running-tests) but not our jar [target](https://github.com/apache/incubator-xtable/blob/main/pom.xml#L63) when publishing.